Document versioning process (#345)
oasislabs/oasis.js: README.md

yarn lint
```

To apply lint fixes:

```
yarn lint:fix
```

### Publishing new npm package versions

This is a multi-package repo; packages are managed by `lerna`.

First, bump the version number in all `package.json`:

```sh
git checkout -b mybranch

> lerna version  # bumps package numbers and creates a commit

Reviewer: ```suggestion
lerna version --no-push  # bumps package numbers and creates a commit
```

Reviewer: Ah, that would only be needed if running the command on master branch

Author: With `git checkout -b mybranch` on the previous branch, an automated push makes sense.

```
yarn lint:fix
```

### Publishing new npm package versions

This is a multi-package repo; packages are managed by `lerna`.

First, bump the version number in all `package.json`:

```sh
git checkout -b mybranch

> lerna version  # bumps package numbers and creates a PR

Reviewer: ```suggestion
lerna version --no-push  # bumps package numbers and creates commit
```

```

> Get the PR approved and submit it. Then pull the newly-update master

Reviewer: ```suggestion
Get the PR approved and merge it. Then pull the newly-updated master
```
